ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1338,6 +1338,16 @@ FCU_DFSS_CHWDC_CSP_RTM_DXZC:
   - filter_alarm
   - run_mode
 
+FCU_SS_RTC_STM_ZTM:
+  description: FCU with return temperature control, supply temperature monitoring, zone temperature monitorning and start/stop control
+  is_canonical: true
+  implements:
+  - FCU
+  - SS
```

Comment:
The use of this is ambigious -- is it the SS control for the fan? Or for another component in the equipment? I would expect to see something like DFSS to indicate fan control.